oslabs-beta/Kubiquity
# Add propTypes

WHAT DOES THIS PR DO? 
1. Adds propTypes package to app. 
2. Add propTypes to React components with props. 

HOW TO TEST THIS PR:
1. Run: `gh pr checkout 31 && npm install && npm run dev`
2. Verify that the app still works. (I think someone knocked out the log data so the table may not load, but I don't think that is the fault of this PR or my work)
3. Verify that the propTypes in the code look correct. 

ADDITIONAL NOTES: 
1. PropTypes is a commonly used library (formally actually part of React) that functions a bit like type scripting—we're saying what data type a prop is supposed to be. 
2. We can get more granular with our propTypes (describe an object's structure while saying it's required in an array), but since our data is still changing a bit, I figured this would be better to add in at a later date.
